tests/integration/components/frost-list-test.js

```diff
@@ -143,11 +143,11 @@ describe(test.label, function () {
     })
 
     it('item 0 is selected', function () {
-      expect(this.$($hook('my-list-item-container', {index: 0})).hasClass('is-selected')).to.eql(true)
+      expect($($hook('my-list-item-container', {index: 0})).hasClass('is-selected')).to.eql(true)
```

Comment:
You are still wrapping these in an extra `$()`.

And it's minor, but since you're going to change this anyway, you could take advantage of `jquery-chai` and write:

`expect($hook('my-list-item-container', {index: 0})).to.have.class('is-selected')`

That way you get a better failure message instead of `false does not deeply equal true`.